pyramid_oereb/lib/readers/real_estate.py

```diff
@@ -57,14 +57,14 @@ def read(self, params, nb_ident=None, number=None, egrid=None, geometry=None):
         plan_for_land_register_config = Config.get_plan_for_land_register_config()
 
         real_estate_view_service = ViewServiceRecord(
-            plan_for_land_register_config.get('reference_wms'),
+            plan_for_land_register_config.get('reference_wms')[Config.get('default_language')],
```

Comment:
perhaps you can get the language from the `params` and then use that language or if it does not exist use the default. Something like what was done for the logo here https://github.com/openoereb/pyramid_oereb/blob/master/pyramid_oereb/lib/readers/extract.py#L150.
Like that you are sure to use the correct language and not only the default language. But first check if this is required.

Reply:
thanks, I will check that :)

Reply:
Yes, I would like some solution suggested by @marionb